org.eclipse.languageserver/src/org/eclipse/languageserver/LSPEclipseUtils.java

```diff
@@ -56,28 +56,31 @@ public static TextDocumentPositionParamsImpl toTextDocumentPosistionParams(URI f
 
 	public static int toEclipseMarkerSeverity(DiagnosticSeverity lspSeverity) {
 		switch (lspSeverity) {
-		case Error: return IMarker.SEVERITY_ERROR;
-		case Warning: return IMarker.SEVERITY_WARNING;
+		case Error:
+			return IMarker.SEVERITY_ERROR;
+		case Warning:
+			return IMarker.SEVERITY_WARNING;
 		default:
 			return IMarker.SEVERITY_INFO;
 		}
 
 	}
-	
-//	public static int toEclipseMarkerSeverity(int lspSeverity) {
-//		switch (lspSeverity) {
-//		case 1: return IMarker.SEVERITY_ERROR;
-//		case 2: return IMarker.SEVERITY_WARNING;
-//		}
-//		return IMarker.SEVERITY_INFO;
-//	}
-	
+
+	//	public static int toEclipseMarkerSeverity(int lspSeverity) {
+	//		switch (lspSeverity) {
+	//		case 1: return IMarker.SEVERITY_ERROR;
+	//		case 2: return IMarker.SEVERITY_WARNING;
+	//		}
+	//		return IMarker.SEVERITY_INFO;
+	//	}
+
 	public static IResource findResourceFor(String uri) {
 		uri = uri.replace("file:///", "file:/");
 		uri = uri.replace("file://", "file:/");
 		IProject project = null;
 		for (IProject aProject : ResourcesPlugin.getWorkspace().getRoot().getProjects()) {
-			if (uri.startsWith(aProject.getLocationURI().toString()) && (project == null || project.getLocation().segmentCount() < aProject.getLocation().segmentCount())) {
+			if (uri.startsWith(aProject.getLocationURI().toString()) && (project == null
```

Comment:
here it would be easier to read if the line break were before of after `&&` so we see the 2 operands.